model/modelindexer/indexer.go

```diff
@@ -265,6 +295,19 @@ func (i *Indexer) processEvent(ctx context.Context, event *model.APMEvent) error
 	atomic.AddInt64(&i.eventsAdded, 1)
 	atomic.AddInt64(&i.eventsActive, 1)
 
+	if serviceName := event.Service.Name + "." + event.Service.Version; serviceName != "." {
```

Comment:
when I originally wrote this I was using the concatenation of these two names below, but stopped doing using this value except as a uniqueness key in the map. So, the `.` is unnecessary now, and can be removed.